### PR TITLE
added check in addRecipe to ensure recipe does not already exist

### PR DESCRIPTION
**Pull request by Grayson34**
removed unnecessary files

### CODE REVIEW

**Review by ryanmercado — APPROVED** [review]
Looks good, we should make tests if we haven't yet though

File: backend/recipe/Recipe.py [hunk 1]
```diff
@@ -31,8 +31,13 @@ def addRecipe(id, recipes):
...
+            for rec in current_recipes:
+                if rec['name'] == recipes[0].name:
+                    recipe_exists = True
 
-            current_recipes.extend(recipes)
+            if not recipe_exists:
+                current_recipes.extend(recipes)
 
             updated_recipes_json = json.dumps(current_recipes, cls=RecipeEncoder)
             cursor.execute('UPDATE recipes SET recipe_lists = ? WHERE id = ?', (updated_recipes_json, id))
```
Comment (cdewitt02) [inline backend/recipe/Recipe.py:43]: I believe that id is now usr_id in the updated DB, all tables now have "usr_id" rather that "id"

**Review by cdewitt02 — CHANGES_REQUESTED** [review]
minor id name change

File: backend/recipe/Recipe.py [hunk 2]
```diff
@@ -106,7 +120,7 @@ def default(self, obj):
 
 #SAMPLE
 # recipe = Recipe(
-#     name='Gerber',
+#     name='Luke',
```
Comment (ryanmercado) [inline backend/recipe/Recipe.py:123]: rip gerber

**Review by WilliamSterner — APPROVED** [review]
good

**Review by cdewitt02 — APPROVED** [review]
Nice!

**Pull request merged**